pkg/reconciler/service/runner.go

```diff
@@ -81,6 +81,7 @@ func (r *runner) reconcile(ctx context.Context, task *reconciler.Task) error {
 	kubeClient, err := k8s.NewKubernetesClient(task.Kubeconfig, r.logger, &k8s.Config{
 		ProgressInterval: r.progressTrackerConfig.interval,
 		ProgressTimeout:  r.progressTrackerConfig.timeout,
+		MaxRetries:       task.ComponentConfiguration.MaxRetries,
 	})
```

Comment:
Ah..sorry, don'T think we have to set this value at all.. the mount of K8s-call retries is not relatd to the MaxRetries for a component. Let's go with the default value by non-setting it